rune/libenclave/internal/runtime/pal/skeleton/tls-server.h

```diff
@@ -0,0 +1,17 @@
+/* *INDENT-OFF* */
+#ifndef TLS_SERVER_H
+#define TLS_SERVER_H
+/* *INDENT-ON* */
+
+#include <stdbool.h>
+
+extern bool tls_server;
+
+#define ENCLAVE_FILENAME "Wolfssl_Enclave.signed.so"
+#define DEFAULT_ADDRESS "/run/rune/ra-tls.sock"
```

Comment:
These two macros are used by tis-server itself only? So not required to be exposed in tis-server.h right?